### ISSUE TITLE
HTTP_HOST error on php-cli

### ISSUE THREAD

**Issue by matthieutirelli** [philkra/elastic-apm-php-agent#32]
When using php cli ( for instance, while running tests ) we may have the ErrorHandler loaded.
It then makes an ErrorException because php-cli does not have any HTTP_HOST defined.

```
ErrorException  : Undefined index: HTTP_HOST
at /.../vendor/philkra/elastic-apm-php-agent/src/Events/EventBean.php:213
    209|                     'hostname' => $_SERVER['SERVER_NAME'] ?? '',
    210|                     'port'     => $_SERVER['SERVER_PORT'] ?? '',
    211|                     'pathname' => $_SERVER['SCRIPT_NAME'] ?? '',
    212|                     'search'   => '?' . (($_SERVER['QUERY_STRING'] ?? '') ?? ''),
  > 213|                     'full' => $http_or_https . '://' . $_SERVER['HTTP_HOST'] . $_SERVER['REQUEST_URI']
    214|                 ],
    215|                 'headers' => [
    216|                     'user-agent' => $headers['User-Agent'] ?? '',
    217|                     'cookie'     => $headers['Cookie'] ?? ''
```

**Comment by matthieutirelli** [comment]
Attempt to resolve in : https://github.com/philkra/elastic-apm-php-agent/pull/33

**Comment by philkra** [comment]
resolved with PR #33 and release `6.4.1`
